.eslintrc.yml

```diff
@@ -3,3 +3,5 @@ extends:
 overrides:
     - files: ['*.js']
       parser: espree
+rules:
+    import/extensions: [error, {js: never, json: always, css: always}]
```

Comment:
why JSON?

Reply:
just copy&pasted from https://github.com/Telefonica/js-toolbox/blob/master/packages/eslint-config/import.yml#L5 and added the css extension